modules/Bio/EnsEMBL/EGPipeline/PipeConfig/VariationMart_conf.pm

```diff
@@ -430,18 +430,18 @@ sub pipeline_analyses {
                             },
       -max_retry_count   => 0,
       -analysis_capacity => 10,
-      -flow_into         => ['CreateDependentTables'],
```

Comment:
I think this was changed to 'CreateMartIndexes' by mistake?

Reply:
Yep, my bad. Thanks, well spotted!